remixer_core/src/main/java/com/google/android/libraries/remixer/annotation/RemixerBindingException.java

```diff
@@ -26,12 +26,11 @@
    * Advice to append at the end of messages.
    */
   private static final String ADVICE =
-      ". Make sure that you're using @RemixerInstance on an instance of this class and that you"
-          + " are not proguarding out this class' binder. Normally adding this to your proguard"
-          + " configuration should help: "
+      ". Make sure that you're using com.google.android.libraries.remixer.annotation. *Method"
+          + " annotations in this class, are not proguarding out this class' binder. Normally"
+          + " adding this to your proguard configuration should help: "
```

Comment:
Also just FYI you should be able to define this for clients via `consumerProguardFiles` in your library's `build.gradle` file.